### PR TITLE
Add more entries to the HNS bestiary.

### PR DESCRIPTION
**Pull request by m-kostrzewa**

### CODE REVIEW

**Review by krdln — APPROVED** [review]

File: doc/developer_guide/HNS_error_bestiary.md [hunk 1]
```diff
@@ -1,15 +1,120 @@
...
+The following document describes common cleanup techniques and error interpretations.
+
+## HNS Decontamination procedures
+
+Before working with HNS, one must realize the risks.
+There are many ways in which HNS can stop working and affect other parts of the system.
+For a developer, it is important to be able to recognize those, which can be recovered 
+from. Below are some techniques which may help.
+
+Always try cleaning using the lowest level procedure. The higher the level, the more
```
Comment (krdln) [inline doc/developer_guide/HNS_error_bestiary.md:13]: The lower lever procedures, ie. with the lower number are actually higher-level, ie. farther from the metal. This is somewhat confusing, but not sure how to reword it. This could be addressed later.

File: doc/developer_guide/HNS_error_bestiary.md [hunk 2]
```diff
@@ -1,15 +1,120 @@
...
+
+### Reproduction:
+
+TODO
+
+## 3. HNS Element not found
+
+### Natural habitat:
+
 1. When attempting to create a transparent docker network
```
Comment (mclapinski) [inline doc/developer_guide/HNS_error_bestiary.md]: I don't get it. This line was under `HNS Unspecified Error` before and it's under `HNS Element not found` now. Was it supposed to be like that?

Comment (krdln) [reply doc/developer_guide/HNS_error_bestiary.md]: I think it belongs to this section, although we should mark it as hypothesis, because we're not sure if that's the cause of this error.

File: doc/developer_guide/HNS_error_bestiary.md (hunk 1 above)
Comment (m-kostrzewa) [reply doc/developer_guide/HNS_error_bestiary.md:13]: Maybe reword it using greek letters? Level Alpha decontamination procedure

The highest level would be Level [REDACTED] decontamination procedure.

Thoughts?

File: doc/developer_guide/HNS_error_bestiary.md (hunk 2 above)
Comment (m-kostrzewa) [reply doc/developer_guide/HNS_error_bestiary.md]: It's HNS Element not found both in linked issue and on confluence on "Issues with Docker and HNS" page. I'm 90% sure that it's Element not found.

File: doc/developer_guide/HNS_error_bestiary.md [hunk 3]
```diff
@@ -1,15 +1,118 @@
...
+
+### Level Gamma decontamination procedure
+
+Use official Microsoft script to cleanup. It will also cleanup some registry entries and do much more.
+
+https://github.com/MicrosoftDocs/Virtualization-Documentation/tree/live/windows-server-container-tools/CleanupContainerHostNetworking
+
+### Level [REDACTED] decontamination procedure
+
+All hope is lost; all we can do is hope that the contamination won't spread to other hosts.
```
Comment (m-kostrzewa) [inline doc/developer_guide/HNS_error_bestiary.md]: > All hope is lost

> all we can do is hope

Deserves an award
